### ISSUE TITLE
Allow specifying alternative configuration file

### ISSUE THREAD

**Issue by tejr** [haxmeister/funtoo-reporter#82]
The default configuration file `/etc/funtoo-report.conf` is sensibly named and located. For development and testing purposes it would be very helpful to be able to specify an alternative path, perhaps with an optional `-c`/`--config` option. Something like this:

```
$ funtoo-report --config ~/.funtoo-report.dev.conf show-json
```

This would apply for all commands, including `config-update`.

If this seems sensible, I think I can make it work. It would tie in very well with issue #81.

**Comment by palica** [comment]
do we want to save the -c --config option for this feature or use it for config-update?

https://github.com/haxmeister/funtoo-reporter/compare/master...docs-and-options#diff-1301376a780a976b5c752ea856d0807fR37

**Comment by ShadowM00n** [comment]
For the record, I adjusted the config-update short name to "-u", both in case this materializes and because it makes better sense anyway.

**Comment by tejr** [comment]
@ShadowM00n: Thanks!

I'll write and make a branch for this feature this weekend.

**Comment by tejr** [comment]
I've implemented this the simplest way I can in #96. I'd like to do it a nicer way with some basic OOP, but that requires checking some things with you guys first.

**Comment by tejr** [comment]
I'm happy with how #96 has turned out with @ShadowM00n's fixes, and this lets me get on with #97, which was my main aim in doing this. Thanks folks.
